lumigator/python/mzai/backend/backend/services/datasets.py

```diff
@@ -242,7 +248,9 @@ def get_dataset_download(self, dataset_id: UUID) -> DatasetDownloadResponse:
                 )
 
             download_urls = []
+            print(f'---> returned {s3_response}')
```

Comment:
remove print log (or alternatively switch to a logger)

Reply:
Sorry. There are a few troubleshooting logs here that shouldn't be :-/